README.md

```diff
@@ -23,81 +23,108 @@ Alternatively  look  here for a [RAML definition](raml/file-upload.raml)
 ### Envelope
 
 #### Create An Envelope
-Creates an envelope and auto generates an Id. Successful response provides in the headers the link  and the envelope Id.
+Creates an envelope and auto generates an Id. The body in the http request must be json. Successful response provides in the headers the link and the envelope Id.
 ```
-POST   	/file-upload/envelopes
+POST   	file-upload/envelopes
 ```
 | Responses    | Status    | Description |
 | --------|---------|-------|
-| Ok  | 201   | Envelope Successfully Created |
-| Bad Request | 400   |  Envelope Not Created |
-| Internal Server Error  | 500   |  INTERNAL_SERVER_ERROR |    
+| Ok  | 201   | Envelope successfully created. |
+| Bad Request | 400   |  Envelope not created. |  
 
 #### Example
 
 Request (POST): localhost:8898/file-upload/envelopes
 
-Body: {}
+Body:
+``` json
+{
+    "callbackUrl": "{user provides}"
```

Comment:
Maybe we can say that this must be absolute url that our service will be able to call and it's up to client to make sure this will be possible. No special validation will be done.